dojson/cli/__init__.py

```diff
@@ -30,8 +34,17 @@
 
 .. code-block:: console
 
-    $ dojson -l marcxml -d marcxml do marc21 do to_marc21 < in.xml > out.xml
-    $ diff in.xml out.xml
+    $ dojson -l marcxml -d marcxml do marc21 do to_marc21 < tests/data/test_1.xml > output.xml
+    $ xmllint --format output.xml | diff - tests/data/test_1.xml
+
+To get a list of fields that are missing mappings use the ``missing`` command:
```

Comment:
BTW, if we are amending docs, then we may want to do some more clean-up work:

- the TOC does not look nice due to section/subsection formatting
- the user talk vs developer talk is not strictly separated

I guess I'll make a separate issue for this.